apps/console/src/auth.html

```diff
@@ -558,7 +558,13 @@
 
         <!-- Start of custom scripts added to the head -->
 
-        <script defer type="text/javascript" src="<%= htmlWebpackPlugin.options.publicPath %>extensions/head-script.js"></script>
+        <script
+            id="head-script"
+            defer
+            type="text/javascript"
+            src="<%= htmlWebpackPlugin.options.publicPath %>extensions/head-script.js"
+            data-page-id="preauth"
```

Comment:
What's the purpose of the `id` and the `data-page-id`.

Reply:
- `id` is used to select the script element. Ex: `const headScript = document.getElementById("head-script");`. 
- `data-page-id` is to identify the current page. `const pageId = headScript.getAttribute("data-page-id");`

Reply:
Cool